flink-connectors/flink-connector-hbase-2.2/src/test/java/org/apache/flink/connector/hbase2/HBaseConnectorITCase.java

```diff
@@ -329,9 +362,13 @@ public void testTableSourceSinkWithDDL() throws Exception {
                         + " FROM "
                         + TEST_TABLE_3
                         + " AS h";
-        Iterator<Row> collected = tEnv.executeSql(query).collect();
+
+        TableResult tableResult3 = tEnv.executeSql(query);
```

Comment:
```suggestion
        TableResult tableResult2 = tEnv.executeSql(query);
```

it's a different method, no need to use `3` I guess.

Reply:
3 is used because of TEST_TABLE_3 has been used.